core/runtime/src/main/java/io/quarkus/runtime/QuarkusApplication.java

```diff
@@ -1,6 +1,17 @@
 package io.quarkus.runtime;
 
+/**
+ * Defines a class as a QuarkusApplication ran by {@link Quarkus}. This is usually used for command mode applications
```

Comment:
I think the first sentence is redundant

Reply:
@geoand Kinda make sense. Changes already reflected